src/main/java/org/folio/circulation/domain/Request.java

```diff
@@ -230,4 +232,19 @@ ItemStatus checkedOutItemStatus() {
   String getDeliveryAddressType() {
     return representation.getString("deliveryAddressTypeId");
   }
+
+  Request changeHoldShelfExpirationDate(DateTime holdShelfExpirationDate) {
+    write(representation, RequestProperties.HOLD_SHELF_EXPIRATION_DATE,
+        holdShelfExpirationDate);
+
+    return this;
+  }
+
+  void removeHoldShelfExpirationDate() {
+    representation.remove(RequestProperties.HOLD_SHELF_EXPIRATION_DATE);
```

Comment:
I might statically import the property name constant to shorten these lines, but that is likely stylistic and certainly does not block this change

Reply:
Shoot! I normally do that. I think it was because I copied the pattern from the position methods above these which includes the class reference. I'll statically import both properties and add it to the PR. It is a small change with little risk.